apps/user-profile/src/app/pages/authentication-page/authentication-overview/authentication-overview.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { MatSlideToggle } from '@angular/material/slide-toggle';
+import { PerunTranslateService, StoreService } from '@perun-web-apps/perun/services';
+
+interface TabItem {
+  icon: string;
+  url: string;
+  label: string;
+  tabName: string;
+}
```

Comment:
Same interface is defined in for settings overview, please define it in one place. Probably perun-models lib.